src/modules/multipart/multipartx.js

```diff
@@ -1,7 +1,7 @@
 import MPSingle from './mpsingle'
 
-export const CHOP_MAX_LEN = 839
-export const FLODATA_MAX_LEN = 1040
+import { FLODATA_MAX_LEN } from '../flo/FLOTransaction'
+export const CHOP_MAX_LEN = 837
```

Comment:
Since `FLODATA_MAX_LEN` is now being imported would it be better to set `CHOP_MAX_LEN = FLODATA_MAX_LEN - X` where `X` is the estimated multi part overhead (does JS even allow arithmetic on an imported value at time of const creation?)

Reply:
I believe JS should support arithmetic on an imported value, but for now I am just going to leave it as is, since this is the code that has been working and I don't want to change anything without full testing.